terraform/modules/iam_role_policy/self_managed_credentials/policy.json

```diff
@@ -8,7 +8,8 @@
                 "iam:GetAccountPasswordPolicy",
                 "iam:ListVirtualMFADevices",
                 "iam:ListUsers",
-                "iam:GetAccountSummary"
+                "iam:GetAccountSummary",
+                "sts:GetFederationToken"
```

Comment:
Let's remove this. We're not quite sure of the impact when the `Resource` is `*`.  It only affects non-admins.  I can live without it (Or when I can't live without it,  I can research the correct Resource to use).